test/support/utils_sort.h

```diff
@@ -386,6 +386,8 @@ run_test(SortTestConfig config,
     // Run tests for USM shared memory (external testing for USM shared memory, once already covered in sycl_iterator.pass.cpp)
     if (config.test_usm_shared)
     {
+        // TODO required later to create temporary policy instance here
+        // and pass it to test_usm with value category like exec parameter
```

Comment:
I would suggest:
`// TODO: Create a temporary policy instance and pass it to test_usm with the needed value category`

Reply:
Fixed